src/NetMQ/Core/Transports/Ipc/IpcAddress.cs

```diff
@@ -48,6 +48,59 @@ public void Resolve(string name, bool ip4Only)
             this.Address = new IPEndPoint(IPAddress.Loopback, hash);
         }
 
+        /// <summary>
+        /// Calculate hash values using the MurmurHash algorithm
+        /// </summary>
+        /// <param name="name">input</param>
+        /// <returns>hash value</returns>
+        private static int GetMurmurHash(string name)
```

Comment:
Wonder if we could move this into a util class, like `Strings`, so it can be reused more easily in future.